python-checks/src/test/java/org/sonar/python/checks/WeakSSLProtocolCheckTest.java

```diff
@@ -33,4 +33,14 @@ public void test() {
   public void test_fallback_import() {
     PythonCheckVerifier.verify("src/test/resources/checks/weakSSLProtocol_fallback_import.py", new WeakSSLProtocolCheck());
   }
+
+  @Test
+  public void test_apigateway() {
+    PythonCheckVerifier.verify("src/test/resources/checks/weakSSLProtocolCheck_apigateway.py", new WeakSSLProtocolCheck());
+  }
+
+  @Test
+  public void test_elasticopensearch() {
+    PythonCheckVerifier.verify("src/test/resources/checks/weakSSLProtocolCheck_ElasticOpenSearch.py", new WeakSSLProtocolCheck());
```

Comment:
The file name format is not consistent. You should find a common format. Please refer to other cases.